### PR TITLE
add relnotes for v1.4.0

### PR DESCRIPTION
**Pull request by dekkagaijin**
```release-note
NONE
```

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,58 @@
...
+* `cosign-linux-pivkey-amd64` releases are now of the form `cosign-linux-pivkey-pkcs11key-amd64` (https://github.com/sigstore/cosign/pull/1052)
+* Releases are now additionally signed using the keyless workflow (https://github.com/sigstore/cosign/pull/1073, https://github.com/sigstore/cosign/pull/1111)
+
+### Enhancements
+
+* Validate the whole attestation statement, not just the predicate (https://github.com/sigstore/cosign/pull/1035)
+* Added the options to replace attestations using `cosign attest --replace` (https://github.com/sigstore/cosign/pull/1039)
+* Added URI to `cosign verify-blob` output (https://github.com/sigstore/cosign/pull/1047)
+* Signatures and certificates created by `cosgin sign` and `cosgin sign-blob` can be output to file using the `--output-signature` and `--output-certificate` flags, respectively (https://github.com/sigstore/cosign/pull/1016, https://github.com/sigstore/cosign/pull/1093, https://github.com/sigstore/cosign/pull/1066, https://github.com/sigstore/cosign/pull/1095)
```
Comment (developer-guy) [inline CHANGELOG.md:17]: *typo here `cosgin`

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,58 @@
...
+* Added the `--signature-digest-algorithm` flag to `cosign verify`, allowing verification of container image signatures which were generated with a non-SHA256 signature algorithm (https://github.com/sigstore/cosign/pull/1071)
+* Builds should now be reproducible (https://github.com/sigstore/cosign/pull/1053)
+* Allows base64 files as `--cert` in `cosign verify-blob` (https://github.com/sigstore/cosign/pull/1088)
+* Kubernetes secrets generated for version >= 1.21 clusters have the immutible bit set (https://github.com/sigstore/cosign/pull/1091)
+* Added `cosign save` and `cosign load` commands to save and upload container images and associated signatures to disk (https://github.com/sigstore/cosign/pull/1094)
+* `cosign sign` will no longer fail to sign private images in keyless mode without `--force` (https://github.com/sigstore/cosign/pull/1116)
+* `cosign verify` now supports signatures stored in files and remote URLs with `--signature` (https://github.com/sigstore/cosign/pull/1068)
+* `cosgin verify` now supports certs stored in files (https://github.com/sigstore/cosign/pull/1095)
```
Comment (developer-guy) [inline CHANGELOG.md:27]: same `cosgin`

**Review by developer-guy — CHANGES_REQUESTED** [review]
just little typos 🙏 looks amazing, thanks @dekkagaijin

File: CHANGELOG.md (hunk 1 above)
Comment (dekkagaijin) [reply CHANGELOG.md]: fixed, ty

File: CHANGELOG.md (hunk 2 above)
Comment (dekkagaijin) [reply CHANGELOG.md]: ditto

**Review by developer-guy — APPROVED** [review]
LGTM 🚀

**Pull request merged**